pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -53,6 +53,8 @@
   private long _numSegmentsQueried = 0L;
   private long _numSegmentsProcessed = 0L;
   private long _numSegmentsMatched = 0L;
+  private long _numConsumingQueried = 0L;
```

Comment:
Can we change this to **_numConsumingSegmentsQueried**? The current name is not clear to me without reading the code.